AdminCore.Constants/PolicyProviderConstants.cs

```diff
@@ -0,0 +1,8 @@
+namespace AdminCore.Constants
+{
+    public class PolicyProviderConstants
+    {
+        public const string PolicyPrefix = "AdminCoreRoles";
+        public const char Separator = '_';
```

Comment:
this doesn't seem or need to be policy provider specific, if this separator is extracted to a generic constant class, the context of string extension can be also made generic.

Reply:
Resolved outside of comments